com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -964,10 +984,10 @@ static void UpgradeAsset(UniversalRenderPipelineAsset asset)
                 asset.k_AssetPreviousVersion = 5;
             }
 
-            if (asset.k_AssetPreviousVersion < 8)
+            if (asset.k_AssetPreviousVersion < 9)
             {
                 // The added feature was reverted, we keep this version to avoid breakage in case somebody already has version 7
```

Comment:
Nit: Comment needs updating. Should the comment be generic to say previous version instead of explicitly saying 7 or 8, etc.